bddl/bddl/config.py

```diff
@@ -24,5 +24,5 @@ def get_domain_filename(domain_name):
 
 
 # MISC
-GROUND_GOALS_MAX_OPTIONS = 20
-GROUND_GOALS_MAX_PERMUTATIONS = 10
+GROUND_GOALS_MAX_OPTIONS = None  # No limit - generates all combinations
```

Comment:
Lets just remove this feature. Let the user worry about it. It's as simple as calling itertools.islice on the generator. Remove the truncated product functions too.